lib/streamlit/DeltaGenerator.py

```diff
@@ -1720,7 +1721,14 @@ def _check_and_convert_to_indices(options, default_values):
                 return None
 
             if not isinstance(default_values, list):
-                default_values = [default_values]
+                if is_type(default_values, "numpy.ndarray") or is_type(
+                    default_values, "pandas.core.series.Series"
+                ):
+                    default_values = list(default_values)
```

Comment:
**Support for `pd` and `np`:** 
This `if` is done before others because calling `if not x` when x is of type `pd.Series()` or `np.array()` throws a `ValueError` exception otherwise.

Also [added protos](https://github.com/streamlit/streamlit/pull/1401/files#diff-5cabc911a858323d334686d7a6a5e5ffR114-R115) of that in the tests.

Reply:
Cool, so I just added what you wrote here to the code for future reference.